### DISCUSSION TITLE
Region IT

### DISCUSSION THREAD

**Discussion by dvdmaz** [PiotrMachowski/Xiaomi-cloud-tokens-extractor#93] [Q&A]
Hello, 
I'm trying to connect to my Xiaomi Miio account, but I've discovered that the account is registered in the Italian region (should be IT).
I've tryied to run the extractor asking for the IT region, even if it's not listed, but it doesn't work... it says "Invalid server provided". I suppose that to implement this the program needs to be aware of the correct address for the Xiaomi server for Italy.
Do you think it's feasible to add it? I'm not experienced in Phyton, I normally write in C#... where can I find the servers list definition?
Thank you, and thanks for sharing your work!

**Comment by PiotrMachowski** [comment]
Have you tried not providing the server? In such case the application tries every server that is known. I suspect that the correct server for Italy is DE.

**Reply by dvdmaz** [reply]
Hello Piotr, and thanks for your quick answer!
Yes, I've tried, the authentication is correct, it list all servers, but none of them shows up a registered device. I suspect that Xiaomi has recently activated a new server for Italy, and I suppose that because if I look at the account summary in Roborock App, it shows Italy as area, not DE, as I expected and you noticed.
I don't know if I can contact Xiaomi for support on this topic, maybe not... When I'll find some time to investigate, I'll try to sniff the traffic originated by the robot to see if it points toward Germany.
Is there a list of Xiamomi known servers IP?
Thank you!
Bye

**Reply by PiotrMachowski** [reply]
You have mentioned Roborock App, it uses a different cloud, so it won't be shown in the Tokens Extractor
